app/views/splash/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div style="display: flex; flex-direction: column; align-items: center; justify-content: center; height: 100vh; background-color: #f7f7f7;">
+  <h1 style="font-size: 2rem; margin-bottom: 20%; color: #424a53; font-size: 50px; font-family: 'Dancing Script', cursive; ">Expense Tracking App</h1>
+  <div style="display: flex;  flex-direction: column; align-items: center; gap: 1.5rem; width: 100%;">
+    <%= link_to "Login", new_user_session_path, style: "padding: 1rem 2rem;  font-size: 1.4rem; background-color: #007bff; color: white; font-family: 'Montserrat', sans-serif; border: none; border-radius: 0.25rem; text-align: center; text-decoration: none; cursor: pointer; transition: background-color 0.3s ease-in-out; width: 20%;", class: "btn-hover" %>
+    <%= link_to "Register", new_user_registration_path, style: "padding: 1rem 2rem;  font-size: 1.4rem; color: #424a53; font-family: 'Montserrat', sans-serif; border: none; border-radius: 0.25rem; text-align: center; text-decoration: none; cursor: pointer; transition: background-color 0.3s ease-in-out; width: 25%; margin-left: 10px;", class: "btn-hover" %>
```

Comment:
- [ ] Kindly consider modifying the design as requested by the previous reviewer and making sure your live demo version is updated. 🙏